#todo: figure out what the parameters should be

scan_to_pointcloud:
  ros_parameters:
    min_dist: 0.1
    neighbor_dist: 0.2

control:
  ros_parameters:
    speed_P_left: 5.0
    speed_P_right: 5.0
    speed_D_left: 1.0
    speed_D_right: 1.0
    speed_I_left: 0.0
    speed_I_right: 0.0
    wheel_radius: 0.3
    max_effort: 4.0
    rate_var: 60.0

magnetometer:
  ros_parameters:

ground_truth:
  ros_parameters:
    x_noise_std_dev: 0.0
    y_noise_std_dev: 0.0
    z_noise_std_dev: 0.0
    roll_noise_std_dev: 0.0
    pitch_noise_std_dev: 0.0
    yaw_noise_std_dev: 0.0
sim_color_detector:
  ros_parameters:

